Sources/Sentry/SentryTracer.m

```diff
@@ -270,14 +270,14 @@ - (void)cancelDeadlineTimer
     self.deadlineTimer = nil;
 }
 
-- (id<SentrySpan>)getActiveSpan
+- (SentrySpan *)getActiveSpan
 {
-    id<SentrySpan> span;
+    SentrySpan *span;
 
     if (self.delegate) {
         @synchronized(_children) {
             span = [self.delegate activeSpanForTracer:self];
-            if (span == nil || span == self || ![_children containsObject:span]) {
+            if (span == nil || ![_children containsObject:span]) {
```

Comment:
Would it make sense to just drop this part of the condition?

Reply:
it works, for sure. 
But since is very common to have the `span==self` is this scenario, is way faster to compare to self then looking up in a list.

Reply:
Ah, that's a good point. I guess I should've made it `span == self.rootSpan`